hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -394,14 +422,9 @@ private void handleNegativeSizedContainers(
         "Container {} has negative size. Please visit Recon's unhealthy " +
             "container endpoint for more details.",
         container.getContainerID());
-    UnhealthyContainers record =
-        ContainerHealthRecords.recordForState(containerHealthStatus,
```

Comment:
need remove handleNegativeSizedContainers() as not used

Reply:
Actually I checked again it is being used by processContainer() method, that runs after we process the existing DB Records.